packages/speech/src/index.js

```diff
@@ -492,6 +432,12 @@ Speech.prototype.createRecognizeStream = function(config) {
     throw new Error('A recognize request requires a configuration object.');
   }
 
+  config = extend(true, {
+    config: {
+      languageCode: 'en-US'
```

Comment:
This is bad. The client library needs to not assume `en-US`. The user must provide this explicitly. (The backend API changed this for a reason.)